webui/react/src/pages/ExperimentList.tsx

```diff
@@ -155,6 +155,8 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
 
   const statesString = useMemo(() => settings.state?.join('.'), [settings.state]);
   const pinnedString = useMemo(() => JSON.stringify(settings.pinned ?? {}), [settings.pinned]);
+  const labelsString = useMemo(() => settings.label?.join('.'), [settings.label]);
+  const usersString = useMemo(() => settings.user?.join('.'), [settings.user]);
```

Comment:
these can be just `settings.label` and `settings.user`? 
and replace them with `labelsString` and `usersString` in [line 175](https://github.com/determined-ai/determined/pull/5299/files#diff-802c2bd28af0e414a3c26dc0c01037f18f453fcc4460f0523a273487587cdcfdR175) and[ line 169](https://github.com/determined-ai/determined/pull/5299/files#diff-802c2bd28af0e414a3c26dc0c01037f18f453fcc4460f0523a273487587cdcfdR169)?
then we can remove `// eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
no, it can't. check the PR description. 

what you are suggesting 

```
const settingsLabels = useMemo(() => settings.label, [settings.label])
```

is no different than

```
const settingsLabels = settings.label
```

if it worked for you, it's probably because you don't have the feature flag on, so you never hit the code in the `ExperimentList` component?


if we wanted to get rid of the `eslint-ignore` we could just unstringify things back within the hook, but it has the same end result and just makes things less clear. see also https://github.com/determined-ai/determined/pull/5187#discussion_r989280857

Reply:
ah i didnt enable feature flag.